### PR TITLE
Button card ui improvement

### PR DESCRIPTION
**Pull request by SubhamB2003**
## Fixes Issue
Issue solved: #1608 

Closes #issue_number

## Checklist

- [x] You've linked this PR to the correct issue.
- [ ] You have checked that the code is working correctly.
- [ ] You ⭐️ the repository!

### CODE REVIEW

**Comment by adity2911** [comment]
share screenshot of the working part of the page

**Comment by SubhamB2003** [comment]
@adity2911 Here it is

image

image

image

I also remove the vertical scroll bar
image

**Comment by Spyware007** [comment]
@SubhamB2003 Can you please fix this Navbar?
It's not matching the UI as well as the Hamburger.
Also, need a full page Navbar!
Can you please fix this?

**Comment by SubhamB2003** [comment]
> @SubhamB2003 Can you please fix this Navbar? It's not matching the UI as well as the Hamburger. Also, need a full page Navbar! Can you please fix this?

Ok, I can fix this but what should I do about the hamburger icon can I change it?

**Comment by adity2911** [comment]
> > @SubhamB2003 Can you please fix this Navbar? It's not matching the UI as well as the Hamburger. Also, need a full page Navbar! Can you please fix this?
> 
> Ok, I can fix this but what should I do about the hamburger icon can I change it?

You can show it to us. If it is alright, it will be approved.

File: src/firebase/auth.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ import { getFirestore } from "firebase/firestore";
...
 
 const firebaseConfig = {
-  apiKey: process.env.REACT_APP_apiKey,
-  authDomain: process.env.REACT_APP_authDomain,
-  projectId: process.env.REACT_APP_projectId,
-  storageBucket: process.env.REACT_APP_storageBucket,
-  messagingSenderId: process.env.REACT_APP_messagingSenderId,
-  appId: process.env.REACT_APP_appId,
-  measurementId: process.env.REACT_APP_measurementId,
+  apiKey: process.env.REACT_APP_APIKEYURL,
```
Comment (adity2911) [inline src/firebase/auth.js:8]: do not change all the names. other people working on this repo will have to face some issues because of it.

Comment (SubhamB2003) [reply src/firebase/auth.js:8]: Yeah sorry for that.

**Comment by SubhamB2003** [comment]
@Spyware007 Hey, I made the changes

image
image

**Comment by Spyware007** [comment]
@PoorviShetty @TANYA-CHAN can you check whether this is working correctly or not?

**Comment by Spyware007** [comment]
@SubhamB2003 when I click on any item of navbar the navbar should be closed?
It's not getting closed!

**Comment by adity2911** [comment]
image
the hamburger seems to be off

**Comment by SubhamB2003** [comment]
> image the hamburger seems to be off

@adity2911 Hey, sorry for that but I am not able to solve this issue can you please only merge the card UI responsiveness issue and remove the vertical scrollbar commit?

**Comment by adity2911** [comment]
sure. You can revert the previous commit and i will merge it

**Comment by SubhamB2003** [comment]
@adity2911 Done

**Pull request merged**